refinery/data_set_manager/test_views.py

```diff
@@ -433,6 +433,40 @@ def test_put_invalid_login(self):
                          '"Only owner may edit attribute order."')
         self.client.logout()
 
+    def test_put_object_id_without_attribute_order(self):
+        updated_attribute_2 = {'id': self.attribute_order_array[2].get('id'),
+                               'rank': 1,
+                               'is_exposed': False,
+                               'is_facet': False,
+                               'is_active': False}
+        self.client.login(username='ownerJane', password='test1234')
+        AttributeOrder.objects.all().delete()
+        response = self.client.put(
+            self.url_root + self.valid_uuid + '/attributes/',
+            updated_attribute_2
+        )
+        self.assertEqual(response.status_code, 404)
+        self.assertEqual(response.content,
+                         '"Could not find attributes to update"')
+        self.client.logout()
+
+    def test_put_object_solr_without_attribute_order(self):
+        updated_attribute_1 = {'solr_field': 'Character_Title_6_3_s',
+                               'rank': 3,
+                               'is_exposed': False,
+                               'is_facet': False,
+                               'is_active': False}
+        self.client.login(username='ownerJane', password='test1234')
```

Comment:
Instead of hard coding, use self.user1 's info

Reply:
the self.user1 info is this - we can't use the object directly because only a hash of the password is stored in the db